### ISSUE TITLE
[Bug]: default_value is ignored

### ISSUE THREAD

**Issue by m9e** [milvus-io/milvus#30048]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: pymilvus-2.3.5
- Deployment mode(standalone or cluster): N/A (client)
- MQ type(rocksmq, pulsar or kafka):    N/A (client)
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus 2.3.5
- OS(Ubuntu or CentOS): osx
- CPU/Memory: lots
- GPU: metal
- Others:
```

### Current Behavior

Note the default value:

```>>> from pymilvus import FieldSchema, Collection, CollectionSchema, DataType, Milvus
>>> z = FieldSchema(name='testint', dtype=DataType.INT64, default_value=31337)
>>> z
{'name': 'testint', 'description': '', 'type': <DataType.INT64: 5>}
>>> z.params
{}
>>> 
```

### Expected Behavior

Created FieldSchema should have a default value.

The downstream effect of this is, of course, that the schema is created incorrectly, applied to the collection incorrectly, and then you get `Field xxx don't match in entities[0]` errors when omitting the field that should use the default_value value.

### Steps To Reproduce

```markdown
1. OSX pymilvus 2.3.2 or 2.3.5 (both tested), REPL, input from above
```

**Comment by m9e** [comment]
To add flavor, https://milvus.io/docs/create_collection.md#parameter-default_value clearly indicates in 2.3.x default_value is accepted and has examples.

**Comment by smellthemoon** [comment]
The default value is actually not available in 2.3.x. It is under development and will be available later. @m9e 
@AnthonyTsu1984 can you help to correct it.

**Comment by m9e** [comment]
Could fix as a doc bug and bring back into docs when the feature exists.
